instrumentation/google.golang.org/grpc/otelgrpc/internal/test/test_utils.go

```diff
@@ -0,0 +1,402 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Can we add a link to the source so we know where to look for updates to this fork?

```suggestion

// Copied from https://github.com/grpc/grpc-go/tree/v1.61.0/interop

```